src/hyppo/dos.asm

```diff
@@ -2725,17 +2749,19 @@ drce_eot_in_filename:
 
         ;; got all characters from this LFN piece
         ;;
-        cpx dos_dirent_longfilename_length
-        bcc drce_piece_didnt_grow_name_length
-        stx dos_dirent_longfilename_length
-        cpx #$3f
-        bcs drce_eot_in_filename2
-
+        lda dos_first_vfat_chunk_in_list_flag
```

Comment:
Please verify this against a filename that is >64 bytes long, to make sure it doesn't do naughty things, like overwrite memory in HYPPO.  The HYPPO simulation environment thingy I made might be a good tool for automatic testing of this and other things.

Reply:
Hmm, I'm not familiar with this HYPPO simulation environment, can you fill me in on this?

Reply:
Ok, I've debugged tonight in xemu with a filename greater than 64 chars. Inside "**dos.asm**", I confirmed that:
- only max 64 chars will be read
- For any entries larger than this, no additional chars will be read and the file will be ignored and we'll move onto the next direntry of interest.

So I'm feeling we're safe on this point.